packages/acs-ui-common/src/telemetry.ts

```diff
@@ -59,7 +59,9 @@ const getTelemetryImplementationHint = (telemetryImplementationHint: _TelemetryI
  * @internal
  */
 export const _getApplicationId = (telemetryImplementationHint: _TelemetryImplementationHint): string => {
-  const highLevelArtifact = 0;
+  // We assume AzureCommunicationLibrary, as we don't currently have any public API to inject otherwise.
```

Comment:
Should it be `@azure/communication-react` here instead of `AzureCommunicationLibrary`?

Reply:
The name/schema is platform agnostic, so we have to use AzureCommunicationLibrary.